### PR TITLE
[GR-1445] Remove Tumor Purity 

### PR DESCRIPTION
**Pull request by avarsava**
Jira ticket or GitHub issue: GR-1445

- [X] Updates CHANGELOG.md
- [Will update wiki when merged] Updates dev docs if applicable

### CODE REVIEW

File: application/dash_application/views/call_ready_wgs.py [hunk 1]
```diff
@@ -178,8 +176,6 @@ def get_merged_wgs_data():
 initial["cutoff_coverage_tumour"] = 80
 cutoff_coverage_normal_label = "Coverage (Normal) minimum"
 initial["cutoff_coverage_normal"] = 30
-cutoff_tumor_purity_label = "Tumor Purity (%) minimum"
```
Comment (slazicoicr) [inline application/dash_application/views/call_ready_wgs.py:181]: You can also remove line 103-109 (code that loads the Purity data that is being removed) and the merge on 142.

File: application/dash_application/views/single_lane_wgs.py [hunk 2]
```diff
@@ -160,8 +158,6 @@ def get_wgs_data():
                 bamqc_df[FASTQC_COL.TotalSequences] *
                 bamqc_df[BAMQC_COL.AverageReadLength] / 1e9)
         , 3)
-    ichorcna_df[special_cols["Purity"]] = round(
```
Comment (slazicoicr) [inline application/dash_application/views/single_lane_wgs.py:163]: You can also remove loading of data (133-138) and merging 163-167)

File: application/dash_application/views/call_ready_wgs.py (hunk 1 above)
Comment (avarsava) [reply application/dash_application/views/call_ready_wgs.py:181]: Done!

File: application/dash_application/views/single_lane_wgs.py (hunk 2 above)
Comment (avarsava) [reply application/dash_application/views/single_lane_wgs.py:163]: Done!

**Review by slazicoicr — APPROVED** [review]

**Review by callunity — APPROVED** [review]

**Pull request merged**